setup.py

```diff
@@ -78,6 +78,7 @@ def run(self):
     "torchmetrics",
     "OpenPatchMiner==0.1.6",
     "pydicom",
+    "openvino-dev==2021.3.0",
```

Comment:
`openvino-dev` installs a lot of dependencies so for ONNX models you can use import from ONNX and install just `openvino` package:

```python
import io
buf = io.BytesIO()

torch.onnx.export(
                model,
                dummy_input.to("cpu"),
                buf,
                opset_version=11,
                export_params=True,
                verbose=True,
                input_names=["input"],
                output_names=["output"],
            )

net = ie.read_network(buf.getvalue(), b"", init_from_buffer=True)
```

Reply:
@dkurt We used to observe for some .onnx models the performance do not match with OV IR models. Is it still an issue? That's actually the reason we want to use MO and convert it to OpenVINO IR.

Reply:
This is not an issue for latest releases. Can you check for this model?

Reply:
This gave me the following error that I don't quite understand, since numpy~=1.17.3 shouldn't conflict with the numpy==1.21.0 requirement. 

The conflict is caused by:
[763](https://github.com/CBICA/GaNDLF/runs/5276145649?check_suite_focus=true#step:4:763)
    gandlf 0.0.14.dev0 depends on numpy==1.21.0
[764](https://github.com/CBICA/GaNDLF/runs/5276145649?check_suite_focus=true#step:4:764)
    openpatchminer 0.1.6 depends on numpy
[765](https://github.com/CBICA/GaNDLF/runs/5276145649?check_suite_focus=true#step:4:765)
    openvino-dev 2021.3.0 depends on numpy~=1.17.3
[766](https://github.com/CBICA/GaNDLF/runs/5276145649?check_suite_focus=true#step:4:766)

[767](https://github.com/CBICA/GaNDLF/runs/5276145649?check_suite_focus=true#step:4:767)
To fix this you could try to:
[768](https://github.com/CBICA/GaNDLF/runs/5276145649?check_suite_focus=true#step:4:768)
1. loosen the range of package versions you've specified
[769](https://github.com/CBICA/GaNDLF/runs/5276145649?check_suite_focus=true#step:4:769)
ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts
[770](https://github.com/CBICA/GaNDLF/runs/5276145649?check_suite_focus=true#step:4:770)
2. remove package versions to allow pip attempt to solve the dependency conflict
[771](https://github.com/CBICA/GaNDLF/runs/5276145649?check_suite_focus=true#step:4:771)

Reply:
I mean you can just install OpenVINO independently (`pip install openvino==2021.4.2`) or in different environment to check that `.onnx` and IR performance is different or not.